### PR TITLE
Feat e2e first install good experience

### PR DESCRIPTION
**Pull request by morarucostel**
Closes 

Updated the chart to include the necessary dependencies. 

#### Changelog

**New**

- Added oauth2-proxy OSS as dependency
- Added tekton pipelines as dependency

**Changed**

- NA

**Removed**

- NA

#### Testing / Reviewing

Tested in our poc environment

### CODE REVIEW

File: examples/e2e-flow-github-auth-values.yaml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      type: ClusterIP
+    zone: untrusted
+  workflow:
+    image:
+      repository: /flow-service-workflow
+      tag: 3.0.8
+    replicaCount: 1
+    service:
+      type: ClusterIP
+    zone: untrusted
```
Comment (tlawrie) [inline examples/e2e-flow-github-auth-values.yaml:49]: Do you think for the example we leave the versions out? that way we only have to keep them updated in the main values.yaml which is hard enough to keep in sync.

Comment (morarucostel) [reply examples/e2e-flow-github-auth-values.yaml:49]: Totally agree with you, I missed that part.
I'll remove the image tags.

Comment (tlawrie) [reply examples/e2e-flow-github-auth-values.yaml:49]: Thanks @morarucostel . Also potentially the `e2e` in the file title.

**Comment by morarucostel** [comment]
@tlawrie done&done.

File: examples/flow-github-auth-values.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+global:
+  database:
+    mongodb:
+      host: bmrg-mongodb
+      secretName: bmrg-mongodb
+  ingress:
+    enabled: true
+    root: /poc-standalone/flow
```
[the remark this answers is not in the document]
Comment (tlawrie) [reply examples/flow-github-auth-values.yaml:8]: And everywhere its referenced like namespaces

**Review by tlawrie — CHANGES_REQUESTED** [review]

**Review by tlawrie — APPROVED** [review]

**Pull request merged**